### PR TITLE
acornfile for django vote app

### PR DESCRIPTION
**Pull request by slayer321**

### CODE REVIEW

File: Acornfile
Comment (samkulkarni20) [inline Acornfile:1]: @slayer321 Are you missing a db container in here?

File: docker-compose.yaml
Comment (samkulkarni20) [inline docker-compose.yaml:1]: Is it needed by the Acorn? If not let's remove it.

**Review by samkulkarni20 — CHANGES_REQUESTED** [review]

File: Acornfile
Comment (slayer321) [reply Acornfile:1]: No, as on the first line of the Acornfile you can see that I'm using services field . So we are using mariadb DB as the service and not directly , that's why we don't have the db field inside our containers field.

File: docker-compose.yaml
Comment (slayer321) [reply docker-compose.yaml:1]: No, not needed. Removing it.

**Review by samkulkarni20 — APPROVED** [review]

**Pull request merged**